docker-for-mac/faqs.md

```diff
@@ -6,112 +6,42 @@ redirect_from:
 
-**Q: Can I switch back and forth between Stable and Edge versions of Docker for Mac?**
+**Q: Can I switch between Stable and Edge versions of Docker Desktop?**
 
-A: Yes, you can switch between versions to try out the Edge releases to see what's new,
-then go back to Stable for other work. However, **you can have only one app
-installed at a time**. Switching back and forth between Stable and Edge apps can
-destabilize your development environment, particularly in cases where you switch
-from a newer (Edge) channel to older (Stable).
-
-For example, containers created with a newer Edge version of Docker for Mac may
-not work after you switch back to Stable because they may have been created
-leveraging Edge features that aren't in Stable yet. Just keep this in mind as
-you create and work with Edge containers, perhaps in the spirit of a playground
-space where you are prepared to troubleshoot or start over.
-
-<font color="#CC3366">To safely switch between Edge and Stable versions be sure
-to save images and export the containers you need, then uninstall the current
-version before installing another. The workflow is described in more detail
-below.</font><br>
-
-#### How to save and restore data
-
-The following procedure can be used to save/restore images and container data,
-for example, if you want to switch between Edge and Stable, or reset your VM
-disk:
-
-1.  Use `docker save -o images.tar image1 [image2 ...]` to save any images you
-    want to keep. (See [save](/engine/reference/commandline/save) in the Docker
-    Engine command line reference.)
-
-2.  Use `docker export -o myContainner1.tar container1` to export containers you
-    want to keep. (See [export](/engine/reference/commandline/export) in the
-    Docker Engine command line reference.)
-
-3.  Uninstall the current app & Install a different version of the app (Stable
-    or Edge), or reset your VM disk.
-
-5.  Use `docker load -i images.tar` to reload previously saved images. (See
-    [load](/engine/reference/commandline/load) in the Docker Engine
-
-6.  Use `docker import -i myContainer1.tar` to create a filesystem image
-    corresponding to previously exported containers. (See
-    [import](/engine/reference/commandline/import) in the Docker Engine
-
-[This
-procedure](https://docs.docker.com/storage/volumes/#backup-restore-or-migrate-data-volumes)
-explains how to backup and restore data volumes.
+A: Yes, you can switch between Stable and Edge versions. You can try out the Edge releases to see what's new, then go back to Stable for other work. However, **you can only have one version of Docker Desktop installed at a time**. For more information, see [Switch between Stable and Edge versions](/docker-for-mac/install/#switch-between-stable-and-edge-versions).
 
```

Comment:
s/Switch/switch

Reply:
The actual section heading is Switch between Stable and Edge versions - https://deploy-preview-9596--docsdocker.netlify.com/docker-for-mac/install/#switch-between-stable-and-edge-versions